torchao/prototype/mx_formats/config.py

```diff
@@ -33,6 +33,11 @@ class MXGemmKernelChoice(Enum):
     CUBLAS = "cublas"
 
 
+class MXFP8Dim1CastKernelChoice(Enum):
```

Comment:
nit: name it to support dim0_dim1 cast if that is added in the future? `MXFP8CastKernelChoice`?